src/test/java/org/tarantool/ClientAsyncOperationsIT.java

```diff
@@ -27,7 +27,7 @@
 
 /**
  * Class with test cases for asynchronous operations
- *
+ * <p>
```

Comment:
I'm a bit afraid about races, so maybe it worth to add a kind of stress test we have for reconnection? I mean, do many ddl / dml operations (in different threads?) and look what will be going on.

Reply:
Okay, I provided a test in `ClientThreadSafeSchemaIT` to be sure different threads can use dml/dql operations without a bad impaction and races between them.